src/testing/e2e/helpers/service-map.ts

```diff
@@ -0,0 +1,69 @@
+import _ from 'lodash';
+
+import { ids } from '~/ui/ids';
+
+import { Preset } from './visit';
+import { welcomeScreen } from './welcome-screen';
+import { E2E as WelcomeScreenE2E } from '~/components/ServiceMapApp/WelcomeScreen';
```

Comment:
You are going to depend on each module/component that is being tested, right?

Reply:
Exactly the goal of this PR.